s4e-web/cypress/integration/map/map-data-search.spec.ts

```diff
@@ -0,0 +1,55 @@
+/// <reference types="Cypress" />
+
+import { Login } from '../../page-objects/auth/auth-login.po';
+import { MapDataSearch } from '../../page-objects/map/map-data-search.po';
+
+before(function () {
+  cy.fixture('users/zkMember.json').as('zkMember');
+});
+
+describe.only('Map Data Search', () => {
+
+  context("Data search for not logged in user", () => {
```

Comment:
Wouldn't be better to call this context 'Not logged in user'?

Reply:
more consistent with other tests